src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,15 +2,23 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Login method called with login: {}", login);
```

Comment:
The log message in this line is good, but it could be more informative. For instance, you could include the method name in the message to provide more context. Here is an example: `logger.info("Method login was called with login: {}", login);`